dev/TabView/TabView.xaml

```diff
@@ -0,0 +1,430 @@
+                                                                Interval="100"
+                                                                Style="{StaticResource RepeatButtonRevealStyle}"
+                                                                Visibility="Collapsed"
+                                                                Content="&#xE76B;"/>
+
+                                                            <ScrollContentPresenter x:Name="ScrollContentPresenter"
+                                                                Grid.Column="1"
+                                                                Padding="{TemplateBinding Padding}" />
+
+                                                            <RepeatButton x:Name="ScrollIncreaseButton"
+                                                                Grid.Column="2"
+                                                                VerticalAlignment="Bottom"
+                                                                HorizontalAlignment="Center"
+                                                                Width="{ThemeResource TabViewItemScrollButtonWidth}"
+                                                                Height="{ThemeResource TabViewItemScrollButtonHeight}"
+                                                                Padding="{ThemeResource TabViewItemScrollButtonPadding}"
+                                                                Background="Transparent"
+                                                                BorderThickness="1"
+                                                                Delay="50"
+                                                                FontFamily="Segoe MDL2 Assets"
+                                                                Interval="100"
+                                                                Style="{StaticResource RepeatButtonRevealStyle}"
+                                                                Visibility="Collapsed"
+                                                                Content="&#xE76C;"/>
+                                                        </Grid>
+                                                    </Border>
+                                                </ControlTemplate>
+                                            </Setter.Value>
+                                        </Setter>
+                                    </Style>
+                                </ScrollViewer.Style>
+
+                                <ItemsPresenter x:Name="TabsItemsPresenter"
+                                    Padding="{TemplateBinding Padding}" />
+
+                            </ScrollViewer>
+
+                            <ContentPresenter
+                                Grid.Column="2"
+                                Content="{TemplateBinding RightCustomContent}"
```

Comment:
Think the spec had different names for these?  As if `TabPlacement` becomes a thing later `Left` and `Right` lose meaning a bit.

Also assuming the area attached to tabs is left for the next iteration?

Reply:
CSS Grids use a Start and End naming scheme to replace Left and Right which looses its context and meaning with Right to Left flow.

You could also use BeforeTabs and AfterTabs

Reply:
Names are definitely not written in stone -- we haven't reviewed the spec yet or gone through API review. These are just preliminary until then!